python-eval-mathrepl/mathrepl.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+# File: mathrepl.py
+
+"""MathREPL, a math expression evaluator using Python's eval() and math."""
+
+import math
+
+__version__ = "1.0"
+__author__ = "Leodanis Pozo Ramos"
+
+ALLOWED_NAMES = {
+    name: obj for name, obj in math.__dict__.items() if "__" not in name
+}
+
+PS1 = "mr>>"
+
+WELCOME = f"""
+MathREPL {__version__}, your Python math expressions evaluator!
+Enter a valid math expression after the prompt "{PS1}".
+Type "help" for more information.
+Type "quit" or "exit" to exit.
```

Comment:
If someone types (literally) `"quit"`, then that string will be printed to the output. I'm not sure what the best way to resolve this is... Maybe it is fine to just assume people will know what to do.